zenoh/src/net/runtime/orchestrator.rs

```diff
@@ -637,6 +647,7 @@ impl Runtime {
         // Must set to nonblocking according to the doc of tokio
         // https://docs.rs/tokio/latest/tokio/net/struct.UdpSocket.html#notes
         socket.set_nonblocking(true)?;
+        socket.set_multicast_ttl_v4(multicast_ttl)?;
```

Comment:
Here we don't join any multicast group. Isn't this a no-op?

Reply:
If I understand multicast properly, only listening for multicast packets requires joining a multicast group, but sending packets only requires sending to the correct IP address, and this is the function used in the scout function [here](https://github.com/eclipse-zenoh/zenoh/blob/25f06bd9e71b4dbed5ba8d75eaa1708693e7b9ec/zenoh/src/scouting.rs#L323C1-L323C70) to create the socket that send out multicast packets, so if the TTL is not set on this socket, then all the scouting messages sent will still have a TTL of 1. I tried taking this line out and saw that packet TTL was still 1.